debian/install

```diff
@@ -22,6 +22,7 @@ host-support/fastboot-gadget.img /var/lib/rpi-sb-provisioner
 host-support/make-boot-image /usr/local/bin
 host-support/rpi-sb-provisioner /etc/default/
 host-support/bootloader.config /var/lib/rpi-sb-provisioner/
+host-support/recovery.bin /var/lib/rpi-sb-provisioner
```

Comment:
How does this handle having 2711 vs 2711 recovery.bin variants ?

Reply:
Currently, it doesn't - this is a 2711 variant, and I expect to come back to this packaging directive once we're ready to release 2712 support.

Reply:
ah, ok. Sorry, I thought that was already there